app.json

```diff
@@ -0,0 +1,279 @@
+      "value": ""
+    },
+
+    "DEV_APP_PORT": {
+      "description": "node port",
+      "required": true,
+      "value": "8090"
+    },
+
+    "OUTPUT_DIR": {
+      "description": "for production build placement",
+      "required": true,
+      "value": "./build"
+    },
+
+    "PUBLIC_DIR": {
+      "description": "for production build placement",
+      "required": true,
+      "value": "./build/client"
+    },
+
+    "ASSETS_DIR": {
+      "description": "for assets directory in production build placement",
+      "required": true,
+      "value": "./build/client/assets"
+    },
+
+    "ASSETS_MAP_FILE": {
+      "description": "for assets directory placement",
+      "required": true,
+      "value": "assets.json"
+    },
+
+    "CAMPAIGN_ID": {
+      "description": "",
+      "required": false,
+      "value": "campaign-id-hash"
+    },
+
+    "DB_HOST": {
+      "description": "to connect to postgres database",
+      "required": true,
+      "value": "https://<changeme>.herokuapp.com"
+    },
+
+    "DB_PORT": {
+      "description": "to connect to postgres database",
+      "required": true,
+      "value": "5432"
+    },
+
+    "DB_NAME": {
+      "description": "database name created in heroku addon",
+      "required": true,
+      "value": "spokeproduction"
+    },
+
+    "DB_TYPE": {
+      "description": "to specify postgres database vs sqlite3",
+      "required": true,
+      "value": "pg"
+    },
+
+    "DB_MIN_POOL": {
+      "description": "for postgres pool on heroku",
+      "required": true,
+      "value": 0
+    },
+
+    "DB_MAX_POOL": {
+      "description": "for postgres pool on heroku",
+      "required": true,
+      "value": 15
+    },
+
+    "DB_USER": {
+      "description": "database user name --> can be anything",
+      "required": true,
+      "value": "spokeadmin"
+    },
+
+    "AUTH0_LOGIN_CALLBACK": {
+      "description": "auth0 login callback url --> this should be filled with your app name and match your auth0 configuration",
+      "required": true,
+      "value": "https://<changeme>.herokuapp.com/login-callback"
+    },
+
+    "AUTH0_LOGOUT_CALLBACK": {
+      "description": "auth0 logout callback url",
+      "required": true,
+      "value": "https://<changeme>.herokuapp.com/logout-callback"
+    },
+
+    "AUTH0_DOMAIN": {
+      "description": "auth0 domain attached to auth0 account",
+      "required": true,
+      "value": "<changeme>.auth0.com"
+    },
+
+    "AUTH0_CLIENT_ID": {
+      "description": "auth0 client id attached to auth0 account",
+      "required": true,
+      "value": "<changeme>"
+    },
+
+    "AUTH0_CLIENT_SECRET": {
+      "description": "auth0 client secret attached to auth0 account",
+      "required": true,
+      "value": "<changeme>"
+    },
+
+    "BASE_URL": {
+      "description": "based on your herokuapp name or domain name",
+      "required": true,
+      "value": "https://<changeme>.herokuapp.com"
+    },
+
+    "GRAPHQL_URL": {
```

Comment:
I pushed against this before, so I'm being a hypocrite but I realize we shouldn't make this another env variable you have to set.  Let's make it optional, and default to `BASE_URL + '/graphql`' (which I think is exactly what @sreynen was suggesting)

Reply:
i'll take it out.